.github/workflows/ci.yml

```diff
@@ -483,3 +493,57 @@ jobs:
                 --accept-flake-config \
                 --extra-experimental-features '"nix-command flakes"' \
                 'github:flox/flox/${{ github.sha }}#packages.${{ matrix.system }}.flox-tests-end2end'
+
+  trigger-flox-installers-workflow:
+    name: "Trigger flox installers workflow"
+    #XXX if: github.ref == 'refs/heads/main' && github.event_name == 'push'
+    runs-on: "ubuntu-22.04-8core"
```

Comment:
question: Would we want to run this workflow on our own runners as well?

Reply:
No, we want to use github's runner since we only need to trigger the workflow in `flox/nix-installers`.